### PR TITLE
Test modify user's informations

### PR DESCRIPTION
**Pull request by ariasnic**
4 test for modifying an existing user and 1 for a non-existent user.

### CODE REVIEW

**Comment by rogargon** [comment]
Have you checked if just admin can change the password and data of other users? Is it possible for a user to change the password of another user?

File: src/main/java/cat/udl/eps/softarch/mynftmkt/config/WebSecurityConfig.java [hunk 1]
```diff
@@ -28,6 +28,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authorizeRequests()
                 .antMatchers(HttpMethod.GET, "/identity").authenticated()
                 .antMatchers(HttpMethod.POST, "/users").anonymous()
+                .antMatchers(HttpMethod.PATCH, "/users/*").authenticated()
```
[the remark this answers is not in the document]
Comment (ariasnic) [reply src/main/java/cat/udl/eps/softarch/mynftmkt/config/WebSecurityConfig.java:31]: Yes, I think it's better that only a user can change their own data. But how can I check that? 
I think that for now every user can change the data of all users. So we have to changer something on the WebSecurityConfig.java file?

[the remark this answers is not in the document]
Comment (rogargon) [reply src/main/java/cat/udl/eps/softarch/mynftmkt/config/WebSecurityConfig.java:31]: For the moment, try to change the security config so just users with role ADMIN can change other users. Some inspiration: https://github.com/UdL-EPS-SoftArch/AgridataHub-API/blob/master/src/main/java/cat/udl/eps/softarch/agridatahub/config/WebSecurityConfig.java

[the remark this answers is not in the document]
Comment (ariasnic) [reply src/main/java/cat/udl/eps/softarch/mynftmkt/config/WebSecurityConfig.java]: I've modified the security config, and added these 2 scenarios : "Modify password without being logged in" and "Modify password being logged in as a user".

**Review by bergacat1 — APPROVED** [review]

**Pull request merged**